core/src/test/java/fr/sncf/osrd/train/StopTests.java

```diff
@@ -1,14 +1,14 @@
 package fr.sncf.osrd.train;
 
 import static fr.sncf.osrd.Helpers.*;
-import static fr.sncf.osrd.railjson.schema.schedule.RJSAllowance.LinearAllowance.MarginType.PERCENTAGE;
+import static fr.sncf.osrd.railjson.schema.schedule.RJSAllowance.MarecoAllowance.MarginType.PERCENTAGE;
```

Comment:
Should we keep both:

```java
import static fr.sncf.osrd.railjson.schema.schedule.RJSAllowance.MarecoAllowance.;
import static fr.sncf.osrd.railjson.schema.schedule.RJSAllowance.LinearAllowance;
```

This way there will be no ambiguity when using `PERCENTAGE`